BranchInfo.props

```diff
@@ -3,8 +3,8 @@
     <MajorVersion>2</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>7</PatchVersion>
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
-    <PreReleaseLabel>servicing</PreReleaseLabel>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <PreReleaseLabel></PreReleaseLabel>
```

Comment:
No need to change the PreReleaseLabel, this one should always be servicing, and it will be used only when StabilizePackageVersion == false.

Reply:
@safern fixed